test/selenium/lib/view/set_up/view.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const View = require('../view');
+const PASSWORDS = require('../../passwords.json');
+
+
+function SetUpView() {
+  View.apply(this, arguments);
+}
+
+SetUpView.prototype = Object.assign({
+
```

Comment:
oh, i didn't know we can define member functions like this as well.  should remember that